### PR TITLE
Add live stream policy

### PR DESCRIPTION
**Pull request by setmao**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**
Add "Live stream policy" select in proposals create/update page.

## Steps to Test This Pull Request
1. Go to "MY PYCONTW".
2. Click on "Create New Talk Proposal (投稿新演講)" or "Create New Tutorial Proposal (投稿新專業課程)".
3. When "recording policy's(議程錄影)" value is False.

## Expected behavior
Should see "Live stream policy (議程直播)" select is below "recording policy (議程錄影)" select.

## More Information
**Screenshots**
Screen Shot 2021-03-02 at 11 46 25 PM

### CODE REVIEW

File: src/static/js/proposals/proposal_form.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        if ($(this).val() == 'False') {
+            $('#div_id_live_stream_policy').fadeIn();
+            $('select#id_live_stream_policy').val('False');
+        } else {
+            $('#div_id_live_stream_policy').fadeOut();
+            $('select#id_live_stream_policy').val('True');
+        }
+    });
+
+})(jQuery);
```
Comment (mattwang44) [inline src/static/js/proposals/proposal_form.js:18]: nit: add a newline at the end of the file

File: src/templates/default/proposals/_proposal_create.html [hunk 2]
```diff
@@ -15,3 +15,7 @@ <h3 class="dashboard-header">
   </div>
 </form>
 {% endblock main-content %}
+
+{% block extra_js %}
+<script src="{% static 'js/proposals/proposal_form.js' %}"></script>
+{% endblock %}
```
Comment (mattwang44) [inline src/templates/default/proposals/_proposal_create.html:21]: nit: add a newline at the end of the file

**Review by mattwang44 — APPROVED** [review]
Nice work! 
I'll merge it after being reviewed by the program team.

**Comment by mattwang44** [comment]
oops it seems the CI failed. Can you also fix the test cases?

File: src/static/js/proposals/proposal_form.js (hunk 1 above)
Comment (setmao) [reply src/static/js/proposals/proposal_form.js]: Resolved in this [commit](https://github.com/pycontw/pycon.tw/pull/983/commits/15e661a6295982954459d1c52e3005d611b79731)

File: src/templates/default/proposals/_proposal_create.html (hunk 2 above)
Comment (setmao) [reply src/templates/default/proposals/_proposal_create.html]: Resolved in this [commit](https://github.com/pycontw/pycon.tw/pull/983/commits/15e661a6295982954459d1c52e3005d611b79731)

**Pull request merged**